client/packages/system/src/Patient/ListView/AppBarButtons.tsx

```diff
@@ -51,6 +61,34 @@ export const AppBarButtons: FC<{ sortBy: SortBy<PatientRowFragment> }> = ({
           {t('button.export', { ns: 'common' })}
         </LoadingButton>
       </Grid>
+      <Modal
+        title=""
+        sx={{ maxWidth: '90%' }}
+        okButton={
+          <DialogButton
+            variant="ok"
+            disabled={patient === undefined}
+            onClick={() => {
+              hideDialog();
+              if (patient) {
+                navigate(patient.id);
```

Comment:
Only if it's not too complicated -- it would be good if the "Okay" button worked when the "Enter" key is pressed. It's the default behaviour in many standard forms, not sure how it would work here. But maybe check if it's done elsewhere? 

But not high priority if it seems like a headache.

Reply:
@mark-prins any suggestions on how to do this?

Reply:
hehe. I thought that too on the create form. not too tricky will see if I can get that in to the other PR